lib/full_text_search/hooks/issue_query_any_searchable.rb

```diff
@@ -1,9 +1,101 @@
 module FullTextSearch
   module Hooks
     module IssueQueryAnySearchable
+      include FullTextSearch::ConditionBuilder
+
       def sql_for_any_searchable_field(field, operator, value)
         # TODO: Implement AND searches across multiple fields.
-        super(field, operator, value)
+        ## TODO List
+        # - filter by match leves('~', '*~')
```

Comment:
If we try to deal with *~, I think we will have to parse the query but we don't want to do that. So we should think of the other way like announcing it to user using Groonga query.